build/version.go

```diff
@@ -5,8 +5,17 @@ import (
 	"strings"
 )
 
-// Version is the current version of siad.
-const Version = "0.6.0"
+const (
+	// Version is the current version of siad.
+	Version = "0.6.1"
```

Comment:
No v0.6.1, bump to v1.0.0 instead. Make sure to replace reference to v0.6.1 with v1.0.0